src/measure_theory/prod.lean

```diff
@@ -753,9 +753,12 @@ variables [second_countable_topology E] [normed_space ℝ E]
 
 lemma integrable.integral_prod_left ⦃f : α × β → E⦄
   (hf : integrable f (μ.prod ν)) : integrable (λ x, ∫ y, f (x, y) ∂ν) μ :=
-integrable.mono hf.integral_norm_prod_left hf.ae_measurable.integral_prod_right' $
-  eventually_of_forall $ λ x, (norm_integral_le_integral_norm _).trans_eq $
-  (norm_of_nonneg $ integral_nonneg_of_ae $ eventually_of_forall $ λ y, (norm_nonneg _ : _)).symm
+begin
+  refine integrable.mono hf.integral_norm_prod_left hf.ae_measurable.integral_prod_right' _,
```

Comment:
This seems to me to be the same proof as before, just in tactic mode, but this works, the other doesn't.

Reply:
Does just wrapping the original term proof in `(... : _)` fix it? (Or wrapping some subexpression?)

Reply:
The only fix I found is to provide explicitly the first `_`, namely `(f (x, y)`. It seems that somehow Lean is now unable to guess some implicit variable (maybe because it has become more difficult with another structure? I imagine that as some point it gives up).